python/python_fastapi_server/routers/wmswcs.py

```diff
@@ -46,6 +46,7 @@ async def handle_wms(req: Request, ):
     # Run adaguc-server
     status, data, headers = adaguc_instance.runADAGUCServer(query_string,
                                                             env=adagucenv,
+                                                            showLogOnError=False,
```

Comment:
Was this change intended?

Reply:
Yes, the logging is in wmswcs.py always displayed, see line 57. It does not need to show it once more when there is an error.